bundle/config/lock.go

```diff
@@ -5,10 +5,6 @@ type Lock struct {
 	// Use a pointer value so that only explicitly configured values are set
 	// and we don't merge configuration with zero-initialized values.
 	Enabled *bool `json:"enabled"`
-
-	// Force acquisition of deployment lock even if it is currently held.
-	// This may be necessary if a prior deployment failed to release the lock.
-	Force bool `json:"force"`
```

Comment:
Let's keep this; force to override warnings vs force to override the lock are different things.

As discussed, let's call this `--force-lock`.